ToyBox/classes/MonkeyPatchin/Vendor/TrashLoot.cs

```diff
@@ -0,0 +1,306 @@
+﻿using HarmonyLib;
```

Comment:
I wouldn't bother reviewing this file yet; it's untouched from the source mod. I'll be doing a lot of work to it.

The file location though - does that work for you?